diff --git a/dist/config/manifest.js b/dist/config/manifest.js
index 3bcdc386f8c487a2197441f8625a0b283263153f..9aace33eaeddffea098e497c02160d8bb1caa113 100644
--- a/dist/config/manifest.js
+++ b/dist/config/manifest.js
@@ -1,6 +1,7 @@
 import { findStylesInModuleGraph } from "../server/collect-styles.js";
 import { VIRTUAL_MODULES } from "./constants.js";
 import { wrapId } from "./vite-utils.js";
+import { fileURLToPath } from "node:url"
 export function manifest(start) {
     let devServer = undefined;
     return {
@@ -13,11 +14,11 @@ export function manifest(start) {
             if (id === VIRTUAL_MODULES.clientViteManifest)
                 return `\0${VIRTUAL_MODULES.clientViteManifest}`;
             if (id === VIRTUAL_MODULES.getClientManifest)
-                return this.resolve(new URL("../server/manifest/client-manifest", import.meta.url).pathname);
+                return this.resolve(fileURLToPath(new URL("../server/manifest/client-manifest", import.meta.url)));
             if (id === VIRTUAL_MODULES.getManifest) {
                 return this.environment.config.consumer === "client"
-                    ? this.resolve(new URL("../server/manifest/client-manifest", import.meta.url).pathname)
-                    : this.resolve(new URL("../server/manifest/ssr-manifest", import.meta.url).pathname);
+                    ? this.resolve(fileURLToPath(new URL("../server/manifest/client-manifest", import.meta.url)))
+                    : this.resolve(fileURLToPath(new URL("../server/manifest/ssr-manifest", import.meta.url)));
             }
             if (id === VIRTUAL_MODULES.middleware) {
                 if (start.middleware)
